NEWS.md

```diff
@@ -11,6 +11,10 @@
   each archive name with a null character (like `find -print0`).  If one or
   more -v arguments are specified, multiple null characters are used to
   separate fields; see the man page for details.
+- tarsnap now accepts --null-output with -x and -t, which causes them to
```

Comment:
I thought this should come after the other `--null-output` entry.  However, that raises the question of whether NEWS.md should be sorted by chronology, or importance.

At the moment, the first item is the strict aliasing, which is probably the least interesting item.

Reply:
Yeah, please re-sort the list in NEWS.md.  (Doesn't need to be exact, just use your judgment.)